learning_journal/templates/layout.jinja2

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
This file is great and all, but none of your other templates are extending from this layout. If you're not gonna use it, kill it.